R/compareScenarios_EDGET.R

```diff
@@ -151,7 +151,7 @@ compareScenarios_EDGET <- function(
     newYaml,
     path = file.path(pathDir, "cs2_main.Rmd"),
     template = system.file(
-      "rmd/compareScenarios_Transport/csEDGET_main.Rmd",
+      "Rmd/compareScenarios_Transport/csEDGET_main.Rmd",
```

Comment:
Sad that we spotted it so late and somehow funny then both at the same time :D Hope it works now for you to render the CompareScenarios!